packages/react/src/FormControl/_FormControlLabel.tsx

```diff
@@ -8,12 +8,14 @@ export type Props = {
    * Whether the label should be visually hidden
    */
   visuallyHidden?: boolean
+  requiredText?: string
+  requiredIndicator?: boolean
```

Comment:
I'm wondering if this might be better handled on `<FormControl>` itself, as to apply the required indicator settings to all labels that exist within. The main con here is that there could be cases where you'd only want to adjust one of the indicators, not all.

Reply:
It makes sense to me to add this to the `FormControl.Label` 🤔 Because that's where the indicator is. I could see users wanting to be able to set a default `requiredText` for an entire form but it looks like there's no `Form` element so that's not really possible. I guess in that instance an engineer would probably create a custom component which inherits from the `FormControl.Label`.

Reply:
Yeah I agree! I think if there's truly value here we could come back to it, but for now I think handling it on `FormControl.Label` is good enough.